### PR TITLE
Created react app

### PR DESCRIPTION
**Pull request by Bolhofner**
very basic setup for future decomposition into composable components

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <meta charset="utf-8">
+    <title>Conduit</title>
+    <!-- Import Ionicon icons & Google Fonts our Bootstrap theme relies on -->
+    <link href="//code.ionicframework.com/ionicons/2.0.1/css/ionicons.min.css" rel="stylesheet" type="text/css">
+    <link href="//fonts.googleapis.com/css?family=Titillium+Web:700|Source+Serif+Pro:400,700|Merriweather+Sans:400,700|Source+Sans+Pro:400,300,600,700,300italic,400italic,600italic,700italic" rel="stylesheet" type="text/css">
+    <!-- Import the custom Bootstrap 4 theme from our hosted CDN -->
+    <link rel="stylesheet" href="//demo.productionready.io/main.css">
+  </head>
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
```
Comment (cmarfia) [inline public/index.html:13]: What is the point of the `<noscript>` tag?

File: public/robots.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
```
Comment (cmarfia) [inline public/robots.txt:2]: What does this file do?

**Review by cmarfia — APPROVED** [review]

File: public/index.html (hunk 1 above)
Comment (Bolhofner) [reply public/index.html:13]: https://chrome.google.com/webstore/detail/noscript/doojmbjmlfjjnbmnoijecmcbfeoakpjm?hl=en

Comment (Bolhofner) [reply public/index.html:13]: The above link is an example of a browser extension which disables javascript. The noscript tag contains a message for users of browsers which do not do javascript or have javascript disabled.

File: public/robots.txt (hunk 2 above)
Comment (Bolhofner) [reply public/robots.txt:2]: The general purpose of the file is managing web crawler traffic to a web page. I believe this configuration tells web crawlers not to crawl any of the pages from the site served with this robots.txt file. This prevents descriptions of the site from being generated by search engine web crawlers, but does not prevent the search engine from indexing links to the site.

**Pull request merged**